nixos/tests/blocky.nix

```diff
@@ -0,0 +1,54 @@
+import ./make-test-python.nix {
+  name = "blocky";
+
+  nodes = {
+    minimalConf = { ... }: {
```

Comment:
I think a single machine is fine. You can drop this one.

Reply:
Again, I think you should drop this additional machine. It is entirely redundant to `biggerConf`.